x-pack/plugins/security_solution/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -141,6 +217,35 @@ export const PolicyList = memo(() => {
           );
         },
       },
+      ...(isProtectionUpdatesEnabled
+        ? [
+            {
+              field: '',
+              name: i18n.translate('xpack.securitySolution.policy.list.deployedVersion', {
+                defaultMessage: 'Deployed Version',
+              }),
+              truncateText: true,
+              render: (policy: PolicyData) => {
+                const [color, displayText, tooltip] = generateDeployedVersionEntry(
```

Comment:
Since there is a lot of logic specific to this. What do you think about moving this out into a new component?

Reply:
My suggestion was to improve readability and keeping the components small, but I'm ok keeping as it is if you think there is not enough logic to put into a separated component. I'm ok with both approaches 🙂